### ISSUE TITLE
New features ideas

### ISSUE THREAD

**Issue by PixelDesigns** [soflyy/oxygen-bugs-and-features#2342]
- Hotkey for functions (copy, paste, delete, add new section / columns etc.)
- Global styles > Headings: Add margin option on H1, H2 etc.

**Comment by bohdanps** [comment]
You can use Hydrogen Pack for hotkeys, works like a charm.

**Comment by PixelDesigns** [comment]
> You can use Hydrogen Pack for hotkeys, works like a charm.

Why should I use an extra plugin just for this?

**Comment by bohdanps** [comment]
Well, you can use this plugin and improve your workflow right now or you can wait years until it's implemented in the core, if it will be implemented at all.

It does much more than hotkeys by the way, ask anyone who works with Oxygen for some time, Hydrogen is a must have.

**Comment by bohdanps** [comment]
> > You can use Hydrogen Pack for hotkeys, works like a charm.
> 
> Why should I use an extra plugin just for this?

Well, you can use this plugin and improve your workflow right now or you can wait years until it's implemented in the core, if it will be implemented at all.

It does much more than hotkeys by the way, ask anyone who works with Oxygen for some time, Hydrogen is a must have.

**Comment by PixelDesigns** [comment]
> > > You can use Hydrogen Pack for hotkeys, works like a charm.
> > 
> > 
> > Why should I use an extra plugin just for this?
> 
> Well, you can use this plugin and improve your workflow right now or you can wait years until it's implemented in the core, if it will be implemented at all.
> 
> It does much more than hotkeys by the way. Ask anyone who works with Oxygen for some time, Hydrogen is a must have.

Yeah I know, but these features would be completely basic. Every WordPress website aims to use few plugins, right? :)
